app/src/main/java/info/nightscout/android/model/medtronicNg/StatusEvent.java

```diff
@@ -0,0 +1,301 @@
+package info.nightscout.android.model.medtronicNg;
```

Comment:
Perhaps we should add an additional package path in here? Maybe info.nightscout.android.model.medtronicNg.test?